src/pages/settings/Wallet/ActivatePhysicalCardPage.tsx

```diff
@@ -34,15 +34,15 @@ type ActivatePhysicalCardPageOnyxProps = {
     cardList: OnyxEntry<Record<string, Card>>;
 };
 
-type ActivatePhysicalCardPageProps = ActivatePhysicalCardPageOnyxProps & StackScreenProps<PublicScreensParamList, typeof SCREENS.TRANSITION_BETWEEN_APPS>;
+type ActivatePhysicalCardPageProps = ActivatePhysicalCardPageOnyxProps & StackScreenProps<SettingsNavigatorParamList, typeof SCREENS.SETTINGS.WALLET.CARD_ACTIVATE>;
```

Comment:
Could you explain this change?

Reply:
ActivatePhysicalCardPage belongs to Settings screens not PublicScreens(and screen `SCREENS.SETTINGS.WALLET.CARD_ACTIVATE` was previously defined in `SettingNavigatorParamList`, but it was not used). Also `SCREENS.TRANSITION_BETWEEN_APPS` provides many redundant props(for example it gives us authTokenType prop which we don't need) and is not meant to be used in this screen, it's just type of route params for some other screen. Instead we can give ActivatePhysicalCardPage it's own type of route props that it needs.

The same applies to other ReportVirtualCardFraudPage, etc.
**TL;DR we shouldn't reuse types from some random screens but provide type for each screen explicitly**

Reply:
Thank you for the explanation!